pkg/object/meshcontroller/worker/ingress.go

```diff
@@ -162,7 +166,7 @@ func (ings *IngressServer) Close() {
 	ings.mutex.Lock()
 	defer ings.mutex.Unlock()
 
-	if ings.Ready() {
+	if ings._ready() {
```

Comment:
need to close the informer?

Reply:
informer is closed in upper layer's worker.